man/repart.d.xml

```diff
@@ -580,6 +580,34 @@
         <para>This option has no effect if the partition already exists.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>Verity=</varname></term>
+
+        <listitem><para>Takes one of <literal>off</literal>, <literal>data</literal> or
+        <literal>hash</literal>. Defaults to <literal>off</literal>. If set to <literal>off</literal> or
+        <literal>data</literal>, the partition is populated with content as specified by
+        <varname>CopyBlocks=</varname> or <varname>CopyFiles=</varname>. If set to <literal>hash</literal>,
+        the partition will be populated with verity hashes from a matching verity data partition. A matching
+        data partition is a partition with <varname>Verity=</varname> set to <literal>data</literal> and the
+        same verity name (as configured with <varname>VerityName=</varname>). If not explicitly configured,
+        the data partition's UUID will be set to the first 128 bits of the verity root hash. Similarly, if
+        not configured, The hash partition's UUID will be set to the final 128 bits of the verity root
+        hash. The verity root hash itself will be included in the output of <command>systemd-repart</command>.
+        </para>
+
+        <para>This option has no effect if the partition already exists.</para>
+
+        <para>Usage of this option in combination with <varname>Encrypt=</varname> is not supported.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>VerityName=</varname></term>
+
+        <listitem><para>Takes a string. This setting is used to find sibling verity partitions for the
+        current verity partition. See the description for <varname>Verity=</varname>.</para></listitem>
+      </varlistentry>
```

Comment:
I think it would make sense to default to the configuration snippet name for VerityMatchKey= if not specified but Verity= is set

Reply:
How would that work? Each of the verity sibling partitions will have a different filename

Reply:
At the moment that doesn't work because you still need to set `Verity=data` for the data partition.

Given that repart now has dropin support, I figure a simple dropin can be used to turn a non verity root partition into a verity data partition while still keeping everything nicely explicit.

Reply:
Yeah, I do not like the suggestion. We should needlessly complicate the matching logic, and should not do match two partitions automatically. So I suggested VerityName=/VerityMatchKey=.